### PR TITLE
*: update offline deployment

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Update deploying a TiDB cluster offline using Ansible
- Update supported versions of TiDB ansible
- Remove TiDB 2.0 GA information in dev, v3.0, v3.1, and v2.1 directories

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/2038
https://github.com/pingcap/docs-cn/pull/2046

### Which version does your change affect? 
dev, v3.0, v2.1, v3.1

### CODE REVIEW

**Comment by juliezhang1112** [comment]
PTAL @liubo0127 Thanks!

File: dev/how-to/deploy/orchestrated/offline-ansible.md [hunk 1]
```diff
@@ -163,13 +167,15 @@ See [Edit the `inventory.ini` file to orchestrate the TiDB cluster](/dev/how-to/
 
 1. You do not need to run the playbook in `ansible-playbook local_prepare.yml`.
 
-2. You can use the `Report` button on the Grafana Dashboard to generate the PDF file. This function depends on the `fontconfig` package and English fonts. To use this function, download the offline installation package, upload it to the `grafana_servers` machine, and install it. This package includes `fontconfig` and `open-sans-fonts`, and only supports the CentOS 7 system.
+2. You can use the `Report` button on the Grafana Dashboard to generate a PDF file. This function depends on the `fontconfig` package and English fonts. To use this function, download the offline installation package, upload it to the `grafana_servers` machine, and install it. This package includes `fontconfig` and `open-sans-fonts`, and only supports the CentOS 7 system.
```
Comment (liubo0127) [inline dev/how-to/deploy/orchestrated/offline-ansible.md]: This item can be deleted in dev and 3.0. It cannot be used after the grafana version is upgraded.

Comment (juliezhang1112) [reply dev/how-to/deploy/orchestrated/offline-ansible.md]: Got it.

Comment (juliezhang1112) [reply dev/how-to/deploy/orchestrated/offline-ansible.md]: Updated. PTAL again~ @liubo0127

Comment (liubo0127) [reply dev/how-to/deploy/orchestrated/offline-ansible.md]: LGTM

**Comment by juliezhang1112** [comment]
Updated. PTAL again. Thanks~ @yikeke

**Pull request merged**